.gitignore

```diff
@@ -12,3 +12,4 @@ echoCoreDocs
 
 packages/echo-core/.yalc
 packages/echo-core/yalc.lock
+.dccache
```

Comment:
What is this? :)

Reply:
This is generated from Snyk extension I have in Visual Studio Code